app/Http/ViewComposers/ProfileComposer.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\ViewComposers;
+
+use Illuminate\View\View;
+// use App\Repositories\Contracts\OptionRepository;
+use App\Eloquent\Option;
+
+class ProfileComposer
+{
+
+    /**
+     * Create a new profile composer.
+     *
+     * @return  void
+     */
+    protected $option;
+
+    public function __construct(
+    )
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.NewlineBeforeOpenBrace</br>**Severity**: error</br>**File**: app/Http/ViewComposers/ProfileComposer.php</br>**Line**: 22</br>The closing parenthesis and the opening brace of a multi-line function declaration must be on the same line